project/Dependencies.scala

```diff
@@ -120,7 +120,8 @@ object Dependencies {
     "org.webjars" % "bootstrap" % bootstrapVersion / "bootstrap.js" minified "bootstrap.min.js" dependsOn "jquery.js",
     "org.webjars.bower" % "momentjs" % s"$momentJsVersion" / s"$momentJsVersion/min/moment-with-locales.js" minified s"$momentJsVersion/min/moment-with-locales.min.js",
     "org.webjars" % "Eonasdan-bootstrap-datetimepicker" % bootstrapDatepickerVersion / s"$bootstrapDatepickerVersion/js/bootstrap-datetimepicker.js"
-      minified s"$bootstrapDatepickerVersion/js/bootstrap-datetimepicker.min.js" dependsOn "bootstrap.js" dependsOn "moment-with-locales.js"
+      minified s"$bootstrapDatepickerVersion/js/bootstrap-datetimepicker.min.js" dependsOn "bootstrap.js" dependsOn "moment-with-locales.js",
+    "org.webjars" % "jquery" % jqueryVersion % Test / s"$jqueryVersion/jquery.js" minified s"$jqueryVersion/jquery.min.js",
```

Comment:
We should add jQuery as a standard JS dependency (not only `Test`). Before release `0.8` I want to remove all JS dependencies, but I want to do this all at once.